launcher/cloudbuild.yaml

```diff
@@ -1,7 +1,7 @@
 substitutions:
   # using this base image for now, because there is an issue causing the newest COS dev
   # image not booting with cs.
-  '_BASE_IMAGE': '' # left empty means using the latest image in the family
+  '_BASE_IMAGE': 'cos-dev-117-18374-0-0' # Use a stable COS version to avoid CS image build issues
```

Comment:
Please do not submit until you get approval from @jkl73 for this change.

Reply:
lgtm, this is the same base image on the last dev build.